giskard/datasets/base/__init__.py

```diff
@@ -507,6 +507,19 @@ def upload(self, client: GiskardClient, project_key: str):
             )
         return dataset_id
 
+    def extract_languages(self):
```

Comment:
We may want to specify the features we want to consider. In some cases, the model only uses a subset of the columns.

Reply:
Yes it makes sense, I added a 'columns' argument (maybe you want to name it 'features'?) and I filter the dataset columns with it.